snowex_db/point_data.py

```diff
@@ -1,7 +1,7 @@
 import logging
 from pathlib import Path
 from typing import List
-
+from timezonefinder import TimezoneFinder
```

Comment:
Does this need a new entry in the `pyproject.toml` dependencies?

Reply:
Yep! It is now (happened in a different branch)